LDAR_Sim/model_code/ldar_sim.py

```diff
@@ -52,11 +52,12 @@ def __init__(self, state, params, timeseries):
             params['working_directory'] + params['leak_file']).iloc[:, 0])
         state['empirical_sites'] = np.array(pd.read_csv(
             params['working_directory'] + params['vent_file']).iloc[:, 0])
-        if isinstance(params['time_offsite'], str):
-            state['offsite_times'] = np.array(pd.read_csv(
-                params['working_directory'] + params['time_offsite']).iloc[:, 0])
-        else:
-            state['offsite_times'] = np.array([params['time_offsite']])
+        if 'time_offsite' in params:
+            if isinstance(params['time_offsite'], str):
+                state['offsite_times'] = np.array(pd.read_csv(
+                    params['working_directory'] + params['time_offsite']).iloc[:, 0])
+            else:
```

Comment:
'time_offsite' parameter should be type 'str' OR type 'list' instead of being type tested.

This might be out of scope for this maybe? If so, merge and we'll clean up later.

Reply:
This is kind of a compatibility thing that I implemented. Totally agree, this just lets us support csvs as well as lists or ints for now. Ideally the database would output a list, which we could use. I think it might not be in scope until Soroush starts bringin gin the database (after v2).

Reply:
OK

Reply:
added as issue #71